11_dictionary/dictionary.rb

```diff
@@ -0,0 +1,30 @@
+class Dictionary
+  def entries
+    @entries ||= {}
```

Comment:
Would prefer doing this in a constructor since all methods are dependent on it, you would avoid doing an if check on each method call